executor/join.go

```diff
@@ -69,6 +78,10 @@ type probeSideTupleFetcher struct {
 }
 
 type probeWorker struct {
+	*hashJoinCtx
+	sessCtx sessionctx.Context
```

Comment:
Move to hashJoinCtx?

Reply:
I'll move it if needed in the following PRs.